src/Internal/BalanceTx/Error.purs

```diff
@@ -89,6 +102,76 @@ instance Show BalanceTxError where
     "ExUnitsEvaluationFailed: " <> printTxEvaluationFailure tx failure
   show e = genericShow e
 
+explainBalanceTxError :: BalanceTxError -> String
+explainBalanceTxError = case _ of
+  BalanceInsufficientError expected actual ->
+    "Insufficient balance. Expected: " <> prettyValue expected
+      <> ", actual: "
+      <> prettyValue actual
+  CouldNotConvertScriptOutputToTxInput ->
+    "Could not convert script output to transaction input"
+  CouldNotGetChangeAddress ->
+    "Could not get change address from the wallet or from the balancer constraints"
+  CouldNotGetCollateral -> "Could not get collateral from wallet"
+  CouldNotGetUtxos ->
+    "Could not get UTxOs from the wallet or the specified source addresses"
+  CollateralReturnError ->
+    "Negative totalCollateral after covering min-utxo-ada requirement."
+      <> "This should be impossible: please report this as a bug to "
+      <>
+        bugTrackerLink
+  CollateralReturnMinAdaValueCalcError coinsPerUtxoUnit txOut ->
+    "Could not calculate minimum Ada for collateral return.\n"
+      <> "Coins per UTxO unit: "
+      <> show coinsPerUtxoUnit
+      <> "\nTransaction output: "
+      <>
+        show txOut
+  ExUnitsEvaluationFailed _ _ ->
+    "Script evaluation failure while trying to estimate ExUnits"
+  InsufficientUtxoBalanceToCoverAsset asset ->
+    "Insufficient UTxO balance to cover asset named "
+      <> asset
+      <> "\nThis should be impossible: please report this as a bug to "
+      <>
+        bugTrackerLink
+  ReindexRedeemersError uir ->
+    "Could not reindex redeemer "
+      <> show uir
+      <> "\nThis should be impossible: please report this as a bug to "
+      <>
+        bugTrackerLink
```

Comment:
```suggestion
      <> bugTrackerLink
```

The formatter sometimes annoyingly does that, I usually fix these manually, please do as well